docker/data-science/notebooks/ch12/0.setup.ipynb

```diff
@@ -282,13 +280,20 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "'blue ray_sessions.gz'\t 'lcd tv_sessions.gz'\n",
-      " bluray_sessions.gz\t  macbook_sessions.gz\n",
-      " dryer_sessions.gz\t  nook_sessions.gz\n",
-      " headphones_sessions.gz   retrotech\n",
-      " ipad_sessions.gz\t 'star trek_sessions.gz'\n",
-      " iphone_sessions.gz\t 'star wars_sessions.gz'\n",
-      " kindle_sessions.gz\t 'transformers dark of the moon_sessions.gz'\n"
+      " ai_pow_search_judgments.txt  'lcd tv_sessions.gz'\n",
```

Comment:
I think this is only supposed to be data in the sessions folder for ch10. Seems like other data is (some of the stack exhange) is being listed, as well. Didn't run the notebook to verify, just noting it looks odd.

Reply:
This is due to local clutter on my machine. Corrected.